Apps/Encounter/src/MapProfiles/HospitalVisitModelProfile.cs

```diff
@@ -31,7 +31,7 @@ public class HospitalVisitModelProfile : Profile
         public HospitalVisitModelProfile()
         {
             this.CreateMap<HospitalVisit, HospitalVisitModel>()
-                .ForMember(dest => dest.Provider, opt => opt.MapFrom(src => src.Clinicians.Where(c => c.RoleDescription == "attending").Select(c => c.DisplayName).SingleOrDefault()))
+                .ForMember(dest => dest.Provider, opt => opt.MapFrom(src => src.Clinicians.Where(c => c.RoleDescription == "Attending Physician").Select(c => c.DisplayName)))
```

Comment:
Why not use FirstOrDefault and not change the model? https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.firstordefault?view=net-7.0